homeassistant/components/google_drive/__init__.py

```diff
@@ -55,11 +54,21 @@ def async_notify_backup_listeners() -> None:
 
     entry.async_on_unload(entry.async_on_state_change(async_notify_backup_listeners))
 
+    entry.runtime_data = GoogleDriveDataUpdateCoordinator(
+        hass, entry=entry, client=client, backup_folder_id=folder_id
+    )
+    await entry.runtime_data.async_config_entry_first_refresh()
+
+    # Set up all platforms for this device/entry.
```

Comment:
Remove the comment as it doesn't add any value. It's also wrong since this isn't a device.

Reply:
Done.